### DISCUSSION TITLE
Can the wave current forces received by a deep-sea bridge pier be modelled using Hydro-UQ

### DISCUSSION THREAD

**Discussion by xiongmengliuliansu** [NHERI-SimCenter/SimCenterCommon#242] [Hydro-UQ]
I would like to simulate the structural response of a deep-sea bridge pile foundation abutment structure in this software, focusing on the study of the wave current force problem, can I use this software to do the calculation?

**Comment by JustinBonus** [comment]
Will depend on a number of things to guarantee validity of calculation. The MPM module in theory can simulate this but likely with very high computational usage, so best fit for full collapse modeling or debris impact inclusion. OpenFOAM-OpenSees (FOAMySees, WiP) is also a good fit for 3D CFD. For a more simple approach, the stochastic wave loading module could likely be generalized to perform a basic deep-sea pile structural response, though the abutment structure might invalidate its mono-pile assumption and geotechnical considerations could require a bit of python code. If you have a semi-empirical model in mind, we can work it into a general Python EVT interface as needed. Lots of ways to approach it though it isn't trivial, do you have a preference?

Message us at bonus@berkeley.edu, nlewis@schnabel-eng.com, and fmckenna@berkeley.edu if you'd like more assistance.
